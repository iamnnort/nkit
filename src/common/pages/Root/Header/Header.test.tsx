import * as React from 'react';
import { mount, ReactWrapper } from 'enzyme';

import HeaderComponent from './Header';

jest.mock('./LangSwitcher/LangSwitcher', () => global.mockComponent('LangSwitcher'));

describe('HeaderComponent', () => {
  let wrapper: ReactWrapper;

  beforeEach(() => {
    wrapper = mount(<HeaderComponent />);
  });

  it(`should not regret`, () => {
    expect(wrapper).toMatchSnapshot();
  });
});
