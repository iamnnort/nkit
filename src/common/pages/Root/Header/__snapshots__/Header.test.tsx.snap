// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderComponent should not regret 1`] = `
.c0 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: end;
  -webkit-justify-content: flex-end;
  -ms-flex-pack: end;
  justify-content: flex-end;
  padding: 15px;
}

<HeaderComponent>
  <styled.header>
    <header
      className="c0"
    >
      <default>
        <MockComponent
          originalComponent="LangSwitcher"
        />
      </default>
    </header>
  </styled.header>
</HeaderComponent>
`;
